### PR TITLE
refactor: reestructure folders organisation

### PR DESCRIPTION
**Pull request by tajespasarela**
EX-6298

### CODE REVIEW

File: packages/x-adapter/src/index.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
-export * from './endpoint-adapter.factory';
+export * from './endpoint-adatpter';
```
Comment (LuisMartinez15) [inline packages/x-adapter/src/index.ts:1]: Fix typo in folder name

```suggestion
export * from './endpoint-adapter';
```

File: packages/x-adapter/src/__tests__/endpoint-adapter.factory.spec.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import { endpointAdapterFactory } from '../endpoint-adapter.factory';
+import { endpointAdapterFactory } from '../endpoint-adatpter/endpoint-adapter.factory';
```
Comment (LuisMartinez15) [inline packages/x-adapter/src/__tests__/endpoint-adapter.factory.spec.ts:1]: I think we should move this file to the `endpoint-adapter/__tests__` folder.

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]
Good job!

File: packages/x-adapter/src/__tests__/endpoint-adapter.factory.spec.ts (hunk 2 above)
Comment (tajespasarela) [reply packages/x-adapter/src/__tests__/endpoint-adapter.factory.spec.ts:1]: done

File: packages/x-adapter/src/index.ts (hunk 1 above)
Comment (tajespasarela) [reply packages/x-adapter/src/index.ts:1]: done

File: packages/x-adapter/src/endpoint-adatpter/index.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export * from './types';
```
[the remark this answers is not in the document]
Comment (tajespasarela) [reply packages/x-adapter/src/endpoint-adatpter/index.ts:1]: done

**Review by LuisMartinez15 — APPROVED** [review]

**Pull request merged**